lib/base-compiler.ts

```diff
@@ -381,7 +381,7 @@ export class BaseCompiler {
     }
 
     supportsObjdump() {
-        return this.objdumperClass !== '';
+        return !!this.objdumperClass;
```

Comment:
The `objdumperClass` will be `undefined` if not being set or being set to an empty string. So the old logic is actually invalid because it can never be `''`.

Reply:
Right! We're slowly trying to fix this kind of type nonsense up. Thanks for finding and fixing.